index.bs

```diff
@@ -535,167 +767,354 @@ To determine whether an |attribute| <dfn>contains a javascript: URL</dfn>:
 
 </div>
 
-## Configuration Processing ## {#configuration-processing}
+## Modify the Configuration ## {#configuration-modify}
+
+The configuration modifier methods are methods on {{Sanitizer}} that modify its configuration.
+They will maintain the validity criteria.
+They return a boolean which informs the caller whether the configuration was modified or not.
+
+<div class=example>
+* `new Sanitizer({elements: ["div"]}).allowElement("p");` &mdash; Returns true. Allows "div" and "p".
+* `new Sanitizer({elements: ["div"]}).removeElement("p");` &mdash; Returns false. Allows "div".
```

Comment:
I think you should flush this out a bit because if you don't assign the instance to a variable it's entirely meaningless and doesn't really allow anything.